models/sap.yml

```diff
@@ -1,11 +1,975 @@
+      - name: empct
+        description: Employment percentage
+      - name: stat2
+        description: Employment status
+      - name: ansvh
+        description: Work contract
+      - name: vdsk1
+        description: Organizational key
+      - name: sname
+        description: Employee's name (sortable, LASTNAME, FIRSTNAME)
+
+
   - name: sap__0fi_gl_10
-    description: Access the totals records from the leading ledger in the new general ledger.
+    description: Extracts the totals records from the leading ledger in the new General Ledger
+    columns:
+      - name: ryear
+        description: Fiscal year
+      - name: activ
+        description: General ledger (transaction attributes)
+      - name: rmvct
+        description: Transaction type
+      - name: rtcur
+        description: Currency key
+      - name: runit
+        description: Base unit of measurement
+      - name: awtyp
+        description: Reference transaction
+      - name: rldnr
+        description: Ledgers in general ledger accounting
+      - name: rrcty
+        description: Record type
+      - name: rvers
+        description: Version
+      - name: logsys
+        description: Logical system
+      - name: racct
+        description: Account number
+      - name: cost_elem
+        description: Cost element
+      - name: rbukrs
+        description: Company code
+      - name: rcntr
+        description: Cost center
+      - name: prctr
+        description: Profit center
+      - name: rfarea
+        description: Functional area
+      - name: rbusa
+        description: Business area
+      - name: kokrs
+        description: Controlling area
+      - name: segment
+        description: Segment for segment reporting
+      - name: scntr
+        description: Sender cost center
+      - name: pprctr
+        description: Partner profit center
+      - name: sfarea
+        description: Partner functional area
+      - name: sbusa
+        description: Business area of the business partner
+      - name: rassc
+        description: Company id of trading partner
+      - name: psegment
+        description: Partner segment for segment reporting
+      - name: faglflext_timestamp
+        description: UTC time stamp in short form (YYYYMMDDhhmmss)
+      - name: currency_type
+        description: Currency type
+      - name: fiscal_period
+        description: Fiscal year/period
+      - name: debit_amount
+        description: Total debit postings
+      - name: credit_amount
+        description: Total credit postings
+      - name: accumulated_balance
+        description: Cumulated balance
+      - name: turnover
+        description: Sales of the period
+
   - name: sap__0fi_gl_14
-    description: Access line items of the leading ledger,  contains all line items that have been extracted from the source system.                              
+    description: Contains the structure to provide new general ledger line item data.
```

Comment:
Request for more details in description of the model.

Reply:
Same note as well about wanting to know the grain of the end model and a test request.

Reply:
Description modified. Based on Edwin's feedback, we will avoid tests here since almost all the data is being provided.